src/settings/CheckoutSettingsForm.js

```diff
@@ -36,34 +54,44 @@ class CheckoutSettingsForm extends React.Component {
     this.props.onSubmit({ other_settings: otherSettings });
   }
 
-  getLastMenu() {
-    const { pristine, submitting } = this.props;
+  renderFooter = () => {
+    const {
+      pristine,
+      submitting,
+    } = this.props;
+
     return (
-      <Button
-        id="clickable-savescanid"
-        type="submit"
-        marginBottom0
-        disabled={(pristine || submitting)}
-      >
-        <FormattedMessage id="ui-circulation.settings.checkout.save" />
-      </Button>
+      <PaneFooter
+        renderEnd={(
+          <Button
+            id="clickable-savescanid"
+            type="submit"
+            buttonStyle="primary paneHeaderNewButton"
+            disabled={(pristine || submitting)}
```

Comment:
```suggestion
            disabled={pristine || submitting}
```
I would get rid of parentheses here.